tests/features/project.feature

```diff
@@ -0,0 +1,24 @@
+@api
+Feature: Project content type.
```

Comment:
Since we are moving to browser tests to test the theme lets remove this and put it in "testProjectTopGroupRender()" test method above.